.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,47 @@
+name: 🐞 Bug
+description: Report an issue to help improve the project.
+title: "[BUG]"
+labels: ["needs-triage"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A brief description of the question or issue, also include what you tried and what didn't work
+    validations:
+      required: true
+  - type: textarea
+    id: Reproduce
+    attributes:
+      label: To Reproduce
+      description: Steps to reproduce this bug
+    validations:
+      required: true
+  - type: textarea
+    id: actual-result
+    attributes:
+      label: Actual Results
+      description: Add your observations/output/log/screenshots if applicable
+    validations:
+      required: false
+  - type: textarea
+    id: expected-result
+    attributes:
+      label: Expected Results
+      description: what is your excepted result?
+    validations:
+      required: false
+  - type: textarea
+    id: extrainfo
+    attributes:
+      label: Additional information
+      description: Is there anything else we should know about this bug? (like Version-Release number of selected component, linux distribution and etc)   
+    validations:
+      required: false
```

Comment:
No newline at EOF.

Reply:
I think it needs fixing:  I've run into too many tools which stumble if there's no newline at the end of the file (which is why both Git itself and GitHub flag this).

Reply:
Fixed